code/script.js

```diff
@@ -316,10 +377,43 @@ const checkMyGuess = (personToCheck) => {
   // 2. Set a Message to show in the win or lose section accordingly
   // 3. Show the win or lose section
   // 4. Hide the game board
+  board.innerHTML = '';
+  winOrLose.style.display = "block";
+  let secretPersonImg = `<img class="cardFinal" src=${secret.img} alt=${secret.name}>`;
+  winOrLoseContainer.insertAdjacentHTML("beforeEnd", secretPersonImg);
```

Comment:
Such a great idea to show the img of the secret person in the winorlosedisplay! Excellent job getting that code to work as well.